assets/json/engines.json

```diff
@@ -0,0 +1,10 @@
+{
+    "BabylonJS": {
+        "divID": "babylonjs",
+        "rootURL": "./engines/babylonjs/?assetUrl={assetUrl}&cameraPosition={cameraPosition}"
```

Comment:
This isn't a rootURL anymore. Consider renaming.

Reply:
I'll just call this `url` instead.